_posts/with-great-data-science.md

```diff
@@ -0,0 +1,105 @@
+# With Great Data Science Comes Great Responsibility
```

Comment:
Please add the following to the top of the file:

```suggestion
---
layout: post
title: {Title of the Post}
subtitle: by {Author of the Post}
---

# With Great Data Science Comes Great Responsibility
```
Change the `{Title of the Post}` and `{Author of the Post}`